core/src/main/java/io/grpc/internal/ServerImpl.java

```diff
@@ -114,14 +114,16 @@
   private final Channelz channelz;
   private final CallTracer serverCallTracer;
 
+  private ServerListener serverListener;
+
   /**
    * Construct a server.
    *
    * @param builder builder with configuration for server
    * @param transportServer transport server that will create new incoming transports
    * @param rootContext context that callbacks for new RPCs should be derived from
    */
-  ServerImpl(
+  public ServerImpl(
```

Comment:
Why this change?

Reply:
Extend package level access to public access so that `io.grpc.servlet.(Servlet)ServerBuilder` can reuse the code. Any better idea to get around it? The only need is to get the `ServerListener` that the `serverImp` instance starts with.

Reply:
I don't see any usage of this in `ServerBuilder`. Can you point me to a line?

Reply:
It was used in `ServerBuilder`, as show in the diff of the commit `get rid of public ServerImpl`, but never mind, I managed to git rid of it in the new commit.